compute/instances.go

```diff
@@ -141,6 +141,8 @@ func (c *InstancesClient) Count(ctx context.Context, input *ListInstancesInput)
 }
 
 func (c *InstancesClient) Get(ctx context.Context, input *GetInstanceInput) (*Instance, error) {
+	var httpError error
```

Comment:
Here I wanted to not make a breaking change,  so I returned the same error as before but also return the machine object.   This matches the behavior that CloudAPI has in Node-Triton.